tgmc.dme

```diff
@@ -1052,8 +1052,8 @@
 #include "code\game\objects\structures\xeno.dm"
 #include "code\game\objects\structures\campaign_structures\campaign_structure.dm"
 #include "code\game\objects\structures\campaign_structures\capture_objectives.dm"
-#include "code\game\objects\structures\campaign_structures\destroy_objectives.dm"
 #include "code\game\objects\structures\campaign_structures\deploy_blockers.dm"
+#include "code\game\objects\structures\campaign_structures\destroy_objectives.dm"
```

Comment:
Did you move this? You should just move it back if you did.

Reply:
havent touched it

Reply:
Its automatic when someone adds it manually instead of ticking